rcon/rcon.py

```diff
@@ -44,7 +44,9 @@
     ALL_ROLES,
     ALL_ROLES_KEY_INDEX_MAP,
     INDEFINITE_VIP_DATE,
+    _get_default_info_dict,
```

Comment:
when we're on that already, some suggestions:
- make this method non-private (remove the underscore)
- rename the method to something more meaningful, e.g.: `new_from_player_name`

what do you think?

Reply:
Updated.